app/assets/stylesheets/ama_layout/layout_components/accordions.scss

```diff
@@ -76,6 +76,105 @@
   }
 }
 
+.accordion-multi-state{
+  @extend .accordion;
+  margin-bottom: $base-margin;
+  background: transparent;
+
+  .accordion-item{
```

Comment:
Okay, so turns out we can't use it because I forgot that .accordion-item is a foundation class that is needed to make them work. This is  also why it was nested inside of the multi accordion class because we only want to change the margin of .accordion-item when it is in a multi state accordion.